.idea/compiler.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we don't normally commit .idea/IDE files do we ?

Reply:
I can never figure it out. The mojo-generated code pre-ignored some files, but then the IDE generates so many more. Some of them are useful, and some are useful but high churn. IDEA have a bit of documentation saying what to ignore and what to commit, and most gets committed.

I just checked main quarkus and there is a `.idea`, but it's minimal. We've committed a picture of Quarkus and a run configuration, but nothing else. So I will stick to that convention and  err on the side of ignoring them all.

Reply:
Slightly smaller changeset now attached - although some .idea files were in the initial commit, so there's still a bit of IDE noise in it from them getting removed.